weasyl/define.py

```diff
@@ -598,24 +598,15 @@ def get_path():
 
 
 def text_price_amount(target):
-    return "%i.%s%i" % (target / 100, "" if target % 100 > 9 else "0", target % 100)
+    return "%.2f" % round(target / 100, 2)
```

Comment:
I’m also pretty sure integers are passed into this function at some point, and define.py doesn’t have a `from __future__ import division`, so you might need to make this `target / 100.0`.